### PR TITLE
OBSDOCS-1356/TRACING-4621: Max's edit

### PR DESCRIPTION
**Pull request by max-cx**
This PR is based on https://github.com/openshift/openshift-docs/pull/82280

Version(s):

4.12, 4.13, 4.14, 4.15, 4.16, 4.17, 4.18

Issue:

https://issues.redhat.com/browse/OBSDOCS-1356

Link to docs preview:

- https://82974--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-configuration-of-instrumentation.html#otel-autoinstrumentation_otel-configuration-of-instrumentation
- https://82974--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-troubleshooting.html#troubleshooting_instrumentation_otel-troubleshoot

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/otel-autoinstrumentation.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+:_mod-docs-content-type: CONCEPT
+[id="otel-autoinstrumentation_{context}"]
+= Auto-instrumentation in the {OTELOperator}
+
+Auto-instrumentation in the {OTELOperator} is the capability to automatically instrument an application without manual code changes. This enables developers and administrators to get observability into applications with minimal effort and changes to the existing codebase.
+
+Auto-instrumentation involves the following processes:
+
+1. The {OTELOperator} injects an init-container, or a side container for Go, to add the instrumentation libraries for the programming language of the instrumented application.
```
Comment (iblancasa) [inline modules/otel-autoinstrumentation.adoc:13]:
```suggestion
1. The {OTELOperator} injects an init-container, or a sidecar container for Go, to add the instrumentation libraries for the programming language of the instrumented application.
```

File: modules/otel-troubleshoot-logging-exporter-stdout.adoc [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="debug-exporter-to-stdout_{context}"]
-= Debug exporter
```
Comment (iblancasa) [inline modules/otel-troubleshoot-logging-exporter-stdout.adoc:7]: Question: should this file be renamed to `otel-troubleshoot-debug-exporter-stdout.adoc`?

File: modules/otel-autoinstrumentation.adoc (hunk 1 above)
Comment (max-cx) [reply modules/otel-autoinstrumentation.adoc:13]: Thank you for spotting. Fixed.

File: modules/otel-troubleshoot-logging-exporter-stdout.adoc (hunk 2 above)
Comment (max-cx) [reply modules/otel-troubleshoot-logging-exporter-stdout.adoc:7]: Thank you. Fixed.

File: modules/otel-troubleshooting-telemetry-data-generation-by-the-instrumentation-libraries.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+. Inspect your application logs for error messages that might indicate that the instrumentation is malfunctioning:
++
+[source,console]
+----
+oc logs <application_pod> -n <workload_project>
+----
+
+. If the application logs contain error messages that indicate that the instrumentation might be malfunctioning, you can install the OpenTelemetry SDK and libraries locally and run your application locally. This way, you can troubleshoot for issues between the instrumentation libraries and your application without {product}.
+
+. Use the Debug Exporter to verify that the telemetry data are reaching the destination OpenTelemetry Collector instance. For more information, see the section about the Debug Exporter.
```
[the remark this answers is not in the document]
Comment (max-cx) [reply modules/otel-troubleshooting-telemetry-data-generation-by-the-instrumentation-libraries.adoc:31]: Unfortunately, the repo rules do not allow putting xref's in module files (this file is a module).

File: modules/otel-troubleshooting-instrumentation-injection-into-your-workload.adoc [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+.Procedure
+
+. Run the following command to verify that the `Instrumentation` object was successfully created:
++
+[source,console]
+----
+oc get instrumentation -n <workload_project> # <1>
+----
+<1> The namespace where the instrumentation is injected for workloads.
```
[the remark this answers is not in the document]
Comment (max-cx) [reply modules/otel-troubleshooting-instrumentation-injection-into-your-workload.adoc:19]: TY, I added this suggestion into your PR.

**Pull request closed without merging**